homeassistant/components/rfxtrx/config_flow.py

```diff
@@ -16,7 +16,6 @@ class ConfigFlow(config_entries.ConfigFlow, domain=DOMAIN):
 
     async def async_step_import(self, import_config=None):
         """Handle the initial step."""
-
         await self.async_set_unique_id(DOMAIN)
-        self._abort_if_unique_id_configured(import_config)
+        self._abort_if_unique_id_configured()
```

Comment:
I would skip this for now. That way it will respect all stuff I'm yaml untill we get option flow in place.

Reply:
Good idea!